### ISSUE TITLE
Add "skip to main content" link

### ISSUE THREAD

**Issue by themeblvd** [themeblvd/jumpstart#410]
Purpose of the skip to main content link:

> "A mechanism is available to bypass blocks of content that are repeated on multiple Web pages."&mdash; WCAG 2.0 Guideline 2.4.1

Link should be placed at the top of the webpage and be the first thing tabbed to. It can be positioned off screen, but when focused (i.e. user hits tab on keyboard), the link appears. It can be positioned absolute off-screen and appear in top left or right corner of screen when it gains focus.
